pkg/storage/stores/indexshipper/compactor/deletion/delete_requests_manager_test.go

```diff
@@ -339,12 +339,6 @@ func TestDeleteRequestsManager_Expired(t *testing.T) {
 				isExpired:           false,
 				nonDeletedIntervals: nil,
 			},
-			expectedDeletionRangeByUser: map[string]model.Interval{
```

Comment:
This is the interesting change in behavior: before, we always created one overarching request of intervals that might be affected but filtering deletes at the load stage means that we know there aren't any potential `Expired` chunks for this user